azure-pipelines.yml

```diff
@@ -29,21 +35,48 @@ jobs:
       timeoutInMinutes: 120
       areaPath: 'DevDiv\Xamarin SDK\Android'
       preBuildSteps:
+        - task: UseDotNet@2
+          displayName: install .NET $(DotNetCoreVersion)
+          inputs:
+            version: $(DotNetCoreVersion)
+            installationPath: /usr/local/share/dotnet/
+          condition: eq(variables['System.JobName'], 'macos')
+        - bash: >
+            export DOTNET_ROOT=/usr/local/share/dotnet/ &&
+            export PATH="$DOTNET_ROOT:~/.dotnet/tools:$PATH" &&
+            curl -L https://dot.net/v1/dotnet-install.sh > dotnet-install.sh &&
+            sh dotnet-install.sh --version $(DotNet6Version) --install-dir $DOTNET_ROOT --verbose &&
+            dotnet --list-sdks &&
+            echo "##vso[task.setvariable variable=PATH]$PATH"
+          displayName: install .NET $(DotNet6Version)
+          condition: eq(variables['System.JobName'], 'macos')
+        - pwsh: |
+            $ProgressPreference = 'SilentlyContinue'
+            Invoke-WebRequest -Uri "https://dot.net/v1/dotnet-install.ps1" -OutFile dotnet-install.ps1
+            & .\dotnet-install.ps1 -Version $(DotNet6Version) -InstallDir "$env:ProgramFiles\dotnet\" -Verbose
+            & dotnet --list-sdks
+          displayName: install .NET $(DotNet6Version)
+          condition: eq(variables['System.JobName'], 'windows')
         - pwsh: |
             dotnet tool uninstall --global Cake.Tool
             dotnet tool install --global Cake.Tool
             dotnet tool install --global boots
-            boots https://aka.ms/xamarin-android-commercial-d16-8-macos
+            dotnet tool install --global api-tools --version 1.1.0-preview.1
+            dotnet tool install --global xamarin.androidbinderator.tool --version 0.4.2
```

Comment:
```
dotnet tool install --global Xamarin.AndroidBinderator.Tool --version 0.4.3
```
that is the reason I use uninstall/install sequence. No need to take care of version - latest will be installed.